protostar/contract_path_resolver/contract_path_resolver.py

```diff
@@ -24,3 +24,7 @@ def contract_path_from_contract_name(self, contract_name: str) -> Path:
         assert contract_paths, f"No contract paths found for {contract_name}!"
 
         return contract_paths[0]
+
+    @property
+    def project_root_path(self):
```

Comment:
We shouldn't expose this one, but probably it won't be necessary if we don't call scarb metadata in hint locals